clusters_calculator.py

```diff
@@ -6,16 +6,18 @@
 import multiprocessing
 
 
-def retrieve_clusters(ne_lat, ne_lng, sw_lat, sw_lng, start_date, end_date, fatal, severe, light, inaccurate, zoom):
-    marker_boxes = divide_to_boxes(ne_lat, ne_lng, sw_lat, sw_lng)
+def retrieve_clusters(**kwargs):
+    marker_boxes = divide_to_boxes(kwargs['ne_lat'], kwargs['ne_lng'], kwargs['sw_lat'], kwargs['sw_lng'])
+    print "marker boxes: "
+    print marker_boxes
     result_futures = []
     logging.info('number of cores: ' + str(multiprocessing.cpu_count()))
     with concurrent.futures.ThreadPoolExecutor(max_workers=multiprocessing.cpu_count()) as executor:
         for marker_box in marker_boxes:
-            markers_in_box = Marker.bounding_box_query(marker_box[0], marker_box[1], marker_box[2], marker_box[3],
-                                                         start_date, end_date, fatal,
-                                                         severe, light, inaccurate).all()
-            result_futures.append(executor.submit(calculate_clusters, markers_in_box, zoom))
+
+            kwargs.update(marker_box)
```

Comment:
Good. Note that if `kwargs` was send to the thread then we'd have to copy the dictionary, since otherwise they'd all only see the latest `marker_box`. But since the queries are done serially it's ok to just update `kwargs`.